src/main/java/writer/WriteToFile.java

```diff
@@ -0,0 +1,19 @@
+package writer;
+
+import dao.Columns;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteToFile implements Writer<Integer, File, String> {
+    @Override
+    public String write(Integer index, File writeTo, String data) {
```

Comment:
```suggestion
    public String write(File fileTo, String data) { // your report should already contain the header
    // we should make this service reusable so that we could write different content to different files as well
```